meta-balena-common/recipes-core/systemd/systemd/resin_update_state_probe

```diff
@@ -88,7 +88,10 @@ if [ "${rpdev}" = "${parent}" ] || valid_device "${part}"; then
 	# UUID is passed by the bootloader in the kernel command line
 	case $label in
 		resin-root*)
-			rdev=$(lsblk -nlo kname,uuid | grep "${ruuid}" | cut -d " " -f1)
+			rdev=$(lsblk -nlo kname,uuid \
+				| grep "$parent" \
```

Comment:
```suggestion
				| grep "${parent}" \
```
NIT, the recommendation by shellcheck is to use brackets even when not required (and the other variable has them).